static/docs/get-started/import-data.md

```diff
@@ -0,0 +1,70 @@
+# Import Data
+
+We've seen how to [push](/doc/get-started/store-data) and
+[pull](/doc/get-started/retrieve-data) data a <abbr>DVC project</abbr>'s
+[remote](/doc/commands-reference/remote). But what if we wanted to integrate a
+dataset or ML model produced in one project into another project (as an
+[external dependency](/doc/user-guide/external-dependencies))?
+
+One way is to download the data (with `wget` or `dvc get`, for example) and use
+`dvc add` to track it, but the connection between projects would be lost. We
+wouldn't be able to tell where the data came from or whether there are new
+versions available.
+
+There's a better alternative (for projects in a Git repository): the
+`dvc import` command. It downloads the data, and creates a
+[DVC-file](/doc/user-guide/dvc-file-format) to track changes in the imported
+data. This is better than using `dvc add` to track downloaded data, because with
+_imports_ we can run `dvc update` to check for changes in the external data
+source before [reproducing](/doc/get-started/reproduce) the
+<abbr>pipeline</abbr> that depends on this data.
```

Comment:
this last part can go after the code snippet?

Reply:
Not sure I understand, please be specific (:

Reply:
Also, I still haven't rewritten the whole expandable section (per https://github.com/iterative/dvc.org/pull/631#pullrequestreview-291330688) but working on it.

Reply:
I simplified this paragraph in my latest commit.